dashboard/src/components/ModelList.tsx

```diff
@@ -42,7 +42,7 @@ export default function ModelList({ queryKey, isLoading, models, paginationProps
                 }
                 return {
                     ...oldData,
-                    items: oldData.items.map((oldModelVersion) => {
```

Comment:
wow, had to google this syntax and learnt there's this thing called Optional Chaining in typescript